lib/npg_pipeline/archive/file/logs.pm

```diff
@@ -0,0 +1,129 @@
+package npg_pipeline::archive::file::logs;
+
+use Moose;
+use Carp;
+use English qw{-no_match_vars};
+
+extends qw{npg_pipeline::base};
+
+our $VERSION = '0';
+
+has 'irods_root' => ( isa => 'Str',
+                      is  => 'rw',
+                    );
+
+sub submit_to_lsf {
+  my ($self, $arg_refs) = @_;
+  my $job_sub = $self->_generate_bsub_command($arg_refs);
+  my $job_id = $self->submit_bsub_command($job_sub);;
+  return ($job_id);
+}
+
+sub _generate_bsub_command {
+  my ($self, $arg_refs) = @_;
+
+  my $irodsinstance = $self->function_list() =~ /gclp/ismx ? q(gclp) : q();
+  my $id_run = $self->id_run();
+
+  my $required_job_completion = $arg_refs->{'required_job_completion'};
+  my $timestamp = $self->timestamp();
+  my $archive_script = q{irods_log_loader.pl};
+  my $job_name_prefix = $archive_script . q{_} . $self->id_run();
+  my $job_name = $job_name_prefix . q{_} . $timestamp;
+
+  my $location_of_logs = $self->make_log_dir( $self->recalibrated_path() );
+  my $bsub_command = q{bsub -q } . $self->lowload_lsf_queue() . qq{ $required_job_completion -J $job_name };
+
+  $bsub_command .=  ( $self->fs_resource_string( {
+    counter_slots_per_job => 1,
+    seq_irods             => $self->general_values_conf()->{default_lsf_irods_resource},
+  } ) ) . q{ };
+
+  $bsub_command .=  qq{-E 'script_must_be_unique_runner -job_name="$job_name_prefix"' };
```

Comment:
I expect that is in place as archival of cram files etc happens in both analysis and archival pipelines which could be running at the same time. As this only runs in the archival pipeline we don't need this.